sample/web-app-compose/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+import org.jetbrains.kotlin.gradle.targets.js.webpack.KotlinWebpackCssSupport
+
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.compose")
+    id("detekt-convention")
+    id("dev.icerock.mobile.multiplatform-resources")
+}
+
+kotlin {
+    js(IR) {
+        browser {
+            webpackTask {
+                cssSupport.enabled = true
```

Comment:
it's should be declared in setup instructions in readme

Reply:
No need in this anymore.